Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Build Stage
```

Comment:
You don't need to copy the Mayhemfile here, since it's parsed from the original clone of the repo.